content/docs/cucumber/reporting.md

```diff
@@ -172,7 +172,6 @@ There are also many third-party plugins:
 * [timestamped-scenarios](https://github.com/moredip/timestamped-scenarios) - Append test run timestamps to each Scenario name as it is being output.
 * [Fivemat](https://github.com/tpope/fivemat) - Cucumber formatter that gives each test file its own line of dots.
 * [Fuubar](https://github.com/martinciu/fuubar-cucumber) - The insta-failing progress bar formatter
-* [Viewcumber](https://github.com/versapay/viewcumber) - Cucumber formatter which generates an HTML website to browse your Scenarios and view screen capture of every single Step.
```

Comment:
Here is the same repo, but from an individual: https://github.com/lpa/viewcumber